src/wrapper/mod.rs

```diff
@@ -80,6 +83,34 @@ pub fn run() -> i32 {
         }
     };
 
+    let tty_fd = STDIN_FILENO;
+    let is_interactive = isatty(tty_fd).unwrap_or(false);
+    if is_interactive {
```

Comment:
All this stuff should be wrapped into a function. And I believe it's better to use that function directly in _run, _run_in_netns, commandline_cmd.

It's good idea to mention in commit message that supervise commands are not implemented yet. (I think it's doable right?)